chromadb/api/__init__.py

```diff
@@ -425,6 +427,7 @@ def get_collection(
     def get_or_create_collection(
         self,
         name: str,
+        configuration: Optional[CollectionConfiguration] = None,
```

Comment:
What are the semantics here if you get() but the configuration is different?

Reply:
We ignore the configuration altogether in the `get` path, but we also don't currently consume the configuration anywhere. As part of another change (https://github.com/chroma-core/chroma/issues/2390) which will make clear that any arguments passed to `get_or_create` are create-path only.